src/modelconfig.hpp

```diff
@@ -150,6 +151,16 @@ class ModelConfig {
          */
     static const char shapeDelimeter = ',';
 
+    /**
+         * @brief custom_loader_options config as map
+         */
+    custom_loader_options_config_t customLoaderOptionsConfigMap;
+
+    /**
+         * @brief custom_loader_options config as string
+         */
+    std::string customLoaderOptionsStr;
```

Comment:
This is to make custom loader implementation simpler. Otherwise, one has to be created from another. Also, while the config file is parsed these two variables are assigned at same place to avoid them going out-of-sync.